### ISSUE TITLE
Request: brew cask formula

### ISSUE THREAD

**Issue by PaulRBerg** [rotki/rotki#550]
## Description

Homebrew has this nice CLI workflow called [cask](https://github.com/Homebrew/homebrew-cask) that makes is possible to administer macOS applications programatically. It is particularly useful for keeping desktop apps in version control.

Here's how to use brew to install the Brave Browser:

```bash
$ brew cask install brave-browser
```

## Specification

It would be nice to have a formula for rotki too:

```bash
$ brew cask install rotki
```

**Comment by tiemonl** [comment]
Hi @LefterisJP, 

I've created a [pull request](https://github.com/Homebrew/homebrew-cask/pull/89880) in Homebrew's [homebrew-cask](https://github.com/Homebrew/homebrew-cask) repository for the current latest version (1.8.0) to add this functionality. If you have any questions, please let me know. Once that pull request is merged, this issue should be closed as I've referenced it from the pull request.

Thanks!

**Comment by LefterisJP** [comment]
Hey @tiemonl thanks a lot for handling this! You are awesome.

Once merged this should make @PaulRBerg happy :smile:

**Comment by LefterisJP** [comment]
The homebrew formula is merged by the homebrew team. I will close the issue as complete. Thank you so much @tiemonl!

Will also add an entry about it in our docs [here](https://github.com/rotki/rotki/blob/develop/docs/installation_guide.rst#osx) to let OSX users know of the homebrew installation option. If you'd like you can also do it @tiemonl since you already did the homebrew formula.

One question from my side on the keeping it up to date. No action would be required on our side, right? Looking at the PR I see:

`url "https://github.com/rotki/rotki/releases/download/v#{version}/rotki-darwin-v#{version}.dmg"`

So long as we follow the same versioning AND naming scheme it will work fine. If not, then we should make a PR to homebrew for adjustments. Is that correct?

**Comment by tiemonl** [comment]
Hey @LefterisJP I can definitely update the docs for you, no problem.

As far as updating it, if the versioning and naming doesn't change, the only thing to change is the 2nd and 3rd line in the cask:
```ruby
  version "1.8.0"
  sha256 "2e4a94d8712c7d9e2ce5d2da3a7a880aa0d617992b9c06af736b86175a0613bb"
```
A PR will have to be opened to update the version variable and the sha256 checksum of the new release dmg. I have no issue updating it in homebrew as I already the environment all set up if you just ping me or something, otherwise it's pretty straight forward following the homebrew docs.
